Dockerfile

```diff
@@ -1,15 +1,11 @@
-FROM mhart/alpine-node:0.10
-MAINTAINER nearForm <info@nearform.com>
+FROM mhart/alpine-node:0.10.38
+MAINTAINER butlerx <butlerx@notthe.cloud>
 
-#RUN apk-install git make gcc g++ python
-RUN apk-install git
-
-RUN mkdir -p /usr/src/app /usr/src/lib /usr/src/config
+RUN apk add --update  git make gcc g++ python postgresql-client && \
+    mkdir -p /usr/src/app
 WORKDIR /usr/src/app
-  
-COPY package.json /usr/src/app/
-COPY lib /usr/src/app/lib/
-COPY config /usr/src/app/config/
-COPY *.js /usr/src/app/
-RUN npm install --production && rm -rf /root/.npm  
-
+ADD . /usr/src/app
+RUN npm install --production \
+    && apk del make gcc g++ python && rm -rf /tmp/* /root/.npm /root/.node-gyp
+EXPOSE 10305
```

Comment:
Question : when deploying multiple instances of a same µs on the same host, this will need to be reworked, right ?